index.html

```diff
@@ -12,10 +12,34 @@
 </head>
 
 <body>
-    <div class="container">
+    
+    <!--Header Section Nav Bar-->
+    <header>
+       
+    <!--<div class="container">-->
+        <div class="container">
+        <nav>   
+            <ul class="navlinks">
+                 <img class="logo" src="img/lambda-black.png" alt="Lambda Logo">
+                <li> <a href="#">Home </a> </li>
+                <li> <a href="about.html">About </a> </li>
+                <li> <a href="#">Products </a> </li>
+                <li> <a href="#">Blog </a> </li>
+                <li> <a href="#">Contact </a> </li>
+            </ul>
+        </nav>
+        <div class="image-container">
+            <img src="img/jumbo.jpg" alt="Jumbo.index image">
+          </div>
+       </div> 
+</header>
+    
    
-        <section class="top-content">
+   
+
```

Comment:
Your getting close to achieving the desired layout keep working on it, you got this!